django/utils/decorators.py

```diff
@@ -15,8 +15,10 @@ def __get__(self, instance, cls=None):
 
 
 def _update_method_wrapper(_wrapper, decorator):
-    # _multi_decorate()'s bound_method isn't available in this scope. Cheat by
-    # using it on a dummy function.
+    """
+    Update the wrapper method to include the attributes from the decorator.
+    """
+
```

Comment:
I think you need to drop this (and other unrelated changes) to keep the patch minimal. 

If you think there's a case for these changes then they should be presented separately

Reply:
Got it I will drop this and other unrelated changes. For example the comments.